drivers/storage/portworx/component/portworx_basic.go

```diff
@@ -46,6 +46,8 @@ const (
 
 var (
 	defaultPxSaTokenExpirationSeconds = int64(12 * 60 * 60)
+	// Accessible for testing purpose
+	RootCaCrtPath = "/var/run/secrets/kubernetes.io/serviceaccount/ca.crt"
```

Comment:
Slightly cleaner way is to keep this variable private and add a function `setRootCaCrtPathForTesting(path string)` to set it.